Modules/Video/BridgeOpenCV/test/itkOpenCVBasicTypeBridgeTest.cxx

```diff
@@ -169,16 +169,16 @@ Size2ConversionTest()
 int
 MatrixConversionTest()
 {
-  constexpr unsigned int NRows = 2;
-  constexpr unsigned int NColumns = 3;
-  using ITKMatrixType = itk::Matrix<double, NRows, NColumns>;
-  using CVMatrixType = cv::Matx<double, NRows, NColumns>;
+  constexpr unsigned int VRows = 2;
+  constexpr unsigned int VColumns = 3;
```

Comment:
`VRows`, `VColumns`? Maybe it should be `NumberOfRows`, `NumberOfColumns`, in this case? These are not formal template parameters anyway... 🤔

Reply:
Those aren't template parameters, and if they were in non-test code then they shouldn't have been changed at all.  I will change them to avoid the `V` and `N` conventions entirely (well, sort of): `NumberOfRows` and `NumberOfColumns`